### ISSUE TITLE
local labels_inst = cluster.cluster(out_instances, labels_segm:eq(15), 1.5)

### ISSUE THREAD

**Issue by gaopeng-eugene** [DavyNeven/fastSceneUnderstanding#2]
label 15 means car. Does it mean to find all semantic class, your method need to search through all labels. Which is quite time consuming?

**Comment by brisker** [comment]
@DavyNeven 
Is CVPPP dataset training code and pretrained code available?

**Comment by DavyNeven** [comment]
Yes, we will upload them soon.

**Comment by brisker** [comment]
@DavyNeven 
![image](https://user-images.githubusercontent.com/13804492/32414924-8462ccb2-c26b-11e7-8b4f-76fe96f7e2b7.png)

what does "concatenated with an x- and y- coordinate map with val-
ues between -1 and 1." mean?
